src/mappers/permissions-mapper.js

```diff
@@ -0,0 +1,26 @@
+import { rawPermissionsSchema } from '../schemas/dal/permissions-schema.js'
+import { createLogger } from '../utils/logger.js'
+
+/**
+ * Takes the raw data and maps it to a more usable format
+ *
+ * @param {Object} raw - The raw response payload from the DAL
+ *
+ * @returns {Object} Formatted permission data
+ */
+
+export const mapPermissions = (raw) => {

```

Comment:
The currently accepted scope for our routes is `BUSINESS_DETAILS:FULL_PERMISSION`

Does this function also need to update the `privileges` to upper case?  Or is this part of our agreed contract with the DAL and therefore the schema should be updated to verify that?

Reply:
Is there a way I can check that the DAL will return `id` and `level` in this format?

Reply:
DAL contract specifies that id and level are lists. Possible values for the 2 lists are in the contract and are all UPPER-CASE.